webapp/src/components/configModal/style.css

```diff
@@ -7,3 +7,7 @@
 .standup-config-tab {
   padding: 12px;
 }
+
+.nav-tabs > li.active > a:focus {
```

Comment:
have we checked the other modals and fixed the issue similarly for them?

Reply:
yes